project/src/pages/homePage/homePage.tsx

```diff
@@ -3,22 +3,27 @@ import {OffersList} from '../../components/offersList/offersList';
 import {useAppDispatch, useAppSelector} from '../../hooks';
 import {useEffect} from 'react';
 import {setOffers} from '../../store/actions';
+import {Offers} from '../../types/offer';
+import {sortOffers} from '../../helpers';
 
-export function HomePage () {
+export function HomePage() {
   const dispatch = useAppDispatch();
   const city = useAppSelector((state) => state.city);
   const offers = useAppSelector((state) => state.offers);
+  const sortOption = useAppSelector((state) => state.sortOption);
 
   useEffect(() => {
     dispatch(setOffers);
   }, [city]);
```

Comment:
React Hook useEffect has a missing dependency: 'dispatch'. Either include it or remove the dependency array